try.go

```diff
@@ -17,12 +17,11 @@
 //		return ..., nil
 //	}
 //
-// This package is not intended for production critical code as quick and easy
-// error handling can occlude critical error handling logic.
+// This package is not intended for production critical code.
```

Comment:
I can imagine that this can be suitable for production code.

It's just that packages that use `try` should only use it within a package, and not expose the fact that it panics to the external API.

Reply:
I added some words to this effect.

Of course, I am myself using it across package boundaries, but as part of a clear, integrated HTTP handling system. And the nice thing about my use is that b/c all the handlers already return errors, its use is optional, transparent, and semantically equivalent.

Anyway, see what you think.

Reply:
SGTM. It's a nice balance of being both a warning, yet also saying that a given use-case is permissible.